pkg/nsx/services/vpc/vpc.go

```diff
@@ -330,6 +330,20 @@ func (s *VPCService) CreatOrUpdatePrivateIPBlock(obj *v1alpha1.VPC, nc common.VP
 	return path, nil
 }
 
+func (s *VPCService) IsSharedVPCNamespaceByNS(ns string) (bool, error) {
+	shared_ns, err := s.getSharedVPCNamespaceFromNS(ns)
```

Comment:
could we use the vpc.tags to check if it's a shared VPC instead of using a getSharedVPCNamespaceFromNS ?

Reply:
No, it is in annotation.